posthog/migrations/0466_datawarehousesavedquery_last_ran_at_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.14 on 2024-09-04 09:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0465_datawarehouse_stripe_account"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="datawarehousesavedquery",
+            name="last_ran_at",
+            field=models.DateTimeField(help_text="The timestamp at which this SavedQuery was last ran.", null=True),
+        ),
+        migrations.AddField(
+            model_name="datawarehousesavedquery",
+            name="last_run_status",
+            field=models.CharField(
+                choices=[
+                    ("Cancelled", "Cancelled"),
+                    ("Completed", "Completed"),
+                    ("Failed", "Failed"),
+                    ("Running", "Running"),
+                    ("Starting", "Starting"),
```

Comment:
We dont have `starting` as a status anywhere else in data warehouse - do we really need this for models? We just set stuff straight to `running`

Reply:
I set starting on the API to have a status between issuing the call to Temporal and temporal running the workflow. My thought was it could give more immediate feedback to users that "stuff is happening" even if Temporal is under pressure. 

That being said, I don't feel that it's very important, and we can always add it later. I can remove this together with the change above.